ARCs/arc-0020.md

```diff
@@ -645,149 +596,6 @@ Calling `get_FIELD` reads and returns the `asset`'s `FIELD` as specified in:
       "returns": {
         "type": "uint64"
       }
-    },
-    {
-      "name": "get_total",
```

Comment:
Love the new asset config endpoint! Wondering if these are needed as well or not? Namely, for situations where you may want to call these within a stateful smart contract? Will a stateful smart contract be able to pull out individual tuple values from `get_asset_config` or not (I've never tried that before)?